event_docs.md.j2

```diff
@@ -30,7 +34,7 @@ This section describes which link types are valid for this event type. For detai
 __Required:__ {{ link_data.required | yes_or_no }}  
 __Legal targets:__ {% if link_data.targets.any_type %}Any{% else %}{% for target_type in link_data.targets.types %}{{ target_type | event_link }}{{ ", " if not loop.last else "" }}{% endfor %}{% endif %}  
 __Multiple allowed:__ {{ link_data.multiple | yes_or_no }}  
-__Description:__ {{ link_data.description }}
+__Description:__ {{ link_data.description }}{% if link_data.experimental %} This link type is experimental and may be removed in a future version of the event.{% endif %}
```

Comment:
It's actually not the link type that is experimental but it points to an event that is experimental, right?
```suggestion
__Description:__ {{ link_data.description }}{% if link_data.experimental %} This link type targets an experimental event and may be removed in a future version of this event.{% endif %}
```